### ISSUE TITLE
Server installer: add command line option for install in basic mode

### ISSUE THREAD

**Issue by soredake** [jellyfin/jellyfin-server-windows#109]
Right now, I can't find an option to force basic mode installation instead of default service mode in Windows installer.

**Comment by anthonylavado** [comment]
I don't think that's exposed as a value, since the installer is custom written in NSIS. Is there a particular reason you'd like that? Is it for automation?

**Comment by soredake** [comment]
> Is there a particular reason you'd like that?

Jellyfin installed in service mode does not have access to external disks, which is dealbreaker for me.

**Comment by anthonylavado** [comment]
Right, when you open the installer and run it through the dialogs, you can choose basic mode. Is that not working? Just trying to understand how I can help 🙂

**Comment by anthonylavado** [comment]
Instead of another edit, just an extra comment. I misread the title and didn't realize you were asking about doing it from the command line.

I don't believe we have any arguments for CLI support. I can add some, but for first installs it's probably best to just go through the GUI dialogs. After that, upgrades keep the same type (basic or service).

Going back to my original question, just out of curiosity, is it to script the install for automation purposes, or...?

**Comment by soredake** [comment]
> In fact, that should be the default.

The default for silent installation seems to be the service mode, I installed jellyfin from winget (it passes only "/S" when doing silent install) and it resulted in service-mode installation.

>is it to script the install for automation purposes

Yes, i install software using winget.
